plenum/common/util.py

```diff
@@ -599,3 +603,9 @@ def check_endpoint_valid(endpoint, required: bool=True):
         raise InvalidEndpointIpAddress(endpoint) from exc
     if not (port.isdigit() and int(port) in range(1, 65536)):
         raise InvalidEndpointPort(endpoint)
+
+
+def getFormattedErrorMsg(msg):
+    msgHalfLength = int(len(msg) / 2)
+    errorLine = "-" * msgHalfLength + "ERROR" + "-" * msgHalfLength
```

Comment:
Why half length, whats wrong with a fixed number of characters or some percentage of screen width. It would look odd when you print 2 error messages one below the other where one of them is significantly different in size